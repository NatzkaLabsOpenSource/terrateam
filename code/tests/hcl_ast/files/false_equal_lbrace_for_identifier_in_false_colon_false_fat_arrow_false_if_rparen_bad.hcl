false = { for foo in false : false => false if )
---
false = { for foo in foo : foo => foo if )
---
false = { for foo in foo : foo => foo if }
---
false = { for foo in foo : foo => foo if ]
---
false = { for foo in foo : foo => foo if *
---
false = { for foo in foo : foo => foo if +
---
false = { for foo in foo : foo => foo if :
