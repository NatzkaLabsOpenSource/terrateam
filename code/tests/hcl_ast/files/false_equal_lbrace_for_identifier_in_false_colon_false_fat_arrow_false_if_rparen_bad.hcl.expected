Expression expected in condition section of object for definition.
